account_migrations/migrate.py

```diff
@@ -0,0 +1,327 @@
+        self.dest_api = dest_api
+        self.dest_app = dest_app
+
+def getKeys(source, destination):
+    source_prefix = key_env_var_prefix_map[source] 
+    dest_prefix = key_env_var_prefix_map[destination]
+    source_api_key = str.format('{}_API_KEY', source_prefix)
+    source_app_key = str.format('{}_APP_KEY', source_prefix)
+    dest_api_key = str.format('{}_API_KEY', dest_prefix)
+    dest_app_key = str.format('{}_APP_KEY', dest_prefix)
+    return AuthKeySet(os.environ[source_api_key], os.environ[source_app_key], os.environ[dest_api_key], os.environ[dest_app_key]) 
+
+def copy_changes(new, old, location, api_key, app_key):
+    print ('copying changes')
+    print ('error: not yet implemented')
+
+def migrate(source, destination, target_screenboard, target_timeboard, target_monitor): 
+    auth_keys = getKeys(source, destination)
+    if target_screenboard:
+        migrate_screenboard(source, destination, target_screenboard, auth_keys)
+    if target_timeboard:
+        migrate_timeboard(source, destination, target_timeboard, auth_keys)
+    if target_monitor:
+        migrate_monitor(source, destination, target_monitor, auth_keys)
+
+"""
+Screenboards
+"""
+def migrate_screenboard(source, destination, target, auth_keys):
+    print (str.format('migrating screenboards ID {} from {} to {}', target, source, destination))
+    dashes_by_name = {}
+
+    target_dash = fetch_screenboard(source, auth_keys.source_api, auth_keys.source_app, target)
+    existing_dashes = fetch_all_screenboards(destination, auth_keys.dest_api, auth_keys.dest_app)
+    print (str.format('found {} dashes in target destination', len(existing_dashes)))
+
+    # create a dictionary that uses the monitor name as its key
+    for dash in existing_dashes:
+        dashes_by_name[dash['title']] = dash
+    dash_exists = has_screenboard_match(target_dash, dashes_by_name)
+
+    if dash_exists:
+        print (str.format('found matching dash!'))
+        existing_dash = dashes_by_name[target_dash['name']]
+        copy_changes(target_dash, existing_dash, destination, auth_keys.dest_api, auth_keys.dest_app)
+        updated_dash = fetch_screenboard(destination, auth_keys.dest_api, auth_keys.dest_app, existing_dash['id'])
+        print (json.dumps(updated_dash))
+    else:
+        print ('no match found, creating screenboard')
+        new_dash = create_screenboard(target_dash, destination, auth_keys.dest_api, auth_keys.dest_app)
+        print(json.dumps(new_dash))
+
+def fetch_screenboard(location, api_key, app_key, dash_id): 
+    from datadog import initialize
+
+    # magic to switch envs
+    os.environ['DATADOG_HOST'] = location
+    options = {
+        'api_key': api_key,
+        'app_key': app_key,
+        }
+
+    initialize(**options)
+    return api.Screenboard.get(dash_id)
+
+def fetch_all_screenboards(location, api_key, app_key):
+    from datadog import initialize
+
+    # magic to switch envs
+    os.environ['DATADOG_HOST'] = location
```

Comment:
this can also be done via:

```
    options = {
        'api_key': foo,
        'app_key': bar,
        'api_host': qux
    }
```

In the api init options hash